test/rulesets/Base/base.jl

```diff
@@ -143,41 +133,35 @@
     end
 
     @testset "muladd(x::$T, y::$T, z::$T)" for T in (Float64, ComplexF64)
-        x, Δx, x̄ = 10randn(T, 3)
-        y, Δy, ȳ = randn(T, 3)
-        z, Δz, z̄ = randn(T, 3)
-        Δk = randn(T)
+        x = 10randn(T)
+        y = randn(T)
+        z = randn(T)
 
-        frule_test(muladd, (x, Δx), (y, Δy), (z, Δz))
-        rrule_test(muladd, Δk, (x, x̄), (y, ȳ), (z, z̄))
+        test_frule(muladd, x, y, z)
+        test_rrule(muladd, x, y, z)
     end
 
     @testset "fma" begin
-        x, Δx, x̄ = 10randn(3)
-        y, Δy, ȳ = randn(3)
-        z, Δz, z̄ = randn(3)
-        Δk = randn()
+        x = 10randn()
+        y = randn()
+        z = randn()
 
-        frule_test(fma, (x, Δx), (y, Δy), (z, Δz))
-        rrule_test(fma, Δk, (x, x̄), (y, ȳ), (z, z̄))
+        test_frule(fma, x, y, z)
+        test_rrule(fma, x, y, z)
     end
 
     @testset "clamp"  begin
-        x̄, ȳ, z̄    = randn(3)
-        Δx, Δy, Δz = randn(3)
-        Δk = randn()
-
         x, y, z = 1., 2., 3.  # to left
```

Comment:
This test set would massively benifit from not using variables and just putting the values in

I think that is true across-the-board.
But here more so